pkg/config/config.go

```diff
@@ -51,12 +51,15 @@ type Config struct {
 	// MultiAZ deploys a cluster across multiple availability zones.
 	MultiAZ bool `env:"MULTI_AZ" sect:"cluster" default:"false"`
 
-	// NoDestroy leaves the cluster running after testing.
-	NoDestroy bool `env:"NO_DESTROY" sect:"cluster" default:"false"`
+	// DestroyClusterAfterTest set to false if you want OCM to clean up the cluster itself after the test completes.
+	DestroyClusterAfterTest bool `env:"DESTROY_CLUSTER" sect:"cluster" default:"true"`
 
 	// Kubeconfig is used to access a cluster.
 	Kubeconfig []byte `env:"TEST_KUBECONFIG" sect:"cluster"`
 
+	// ClusterExpiryInMinutes is how long before a cluster expires and is deleted by OSD
+	ClusterExpiryInMinutes int64 `env:"CLUSTER_EXPIRY_IN_MINUTES" sect:"cluster" default:"210"`
```

Comment:
The documentation for ExpiryInHours says 5 hours, yet this would be 3.5 hours. Which is correct?

Reply:
I meant for this to be 3.5 hours, so I'll update the docs. I'm a little worried about quota issues and figure we can configure it differently if we'd like.